src/issues-list.html

```diff
@@ -1,17 +1,27 @@
 
 <link rel="import" href="shared-styles.html">
+<link rel="import" href="issue-priority.html">
 <link rel="import" href="../bower_components/vaadin-grid/vaadin-grid.html">
+<link rel="import" href="../bower_components/vaadin-grid/vaadin-grid-sorter.html">
 <link rel="import" href="../bower_components/iron-ajax/iron-ajax.html">
 
+
 <dom-module id="issues-list">
   <template>
+    <style>
+    </style>
+
     <!-- Fetch an array of users to be shown in the grid -->
     <iron-ajax auto url="/dummy/issues.json" handle-as="json" last-response="{{issues}}"></iron-ajax>
     <!-- The array is set as the <vaadin-grid>'s "items" property -->
     <vaadin-grid items='[[issues]]'>
```

Comment:
Style nitpick alarm!

We prefer using double-quotes in HTML attributes: `items="[[issues]"` and single-quotes in Javascript strings.